Source/WebCore/platform/LengthPoint.h

```diff
@@ -32,36 +32,27 @@ namespace WebCore {
 struct BlendingContext;
 
 struct LengthPoint {
-public:
     LengthPoint()
     {
     }
-    
+
     LengthPoint(Length x, Length y)
-        : m_x(WTFMove(x))
-        , m_y(WTFMove(y))
+        : x(x)
+        , y(y)
     {
     }
```

Comment:
Do we actually need these constructors? Pretty sure they're default.

Reply:
We do not. The only slight benefit that this has is that if another class has a LengthPoint member variable in it, I believe you would need to use brackets instead of parentheses in an initializer list such as: `m_lengthPoint{x, y}` instead of `m_lengthPoint(x, y)`. The initialization with the parentheses would try to call a constructor that specifically takes 2 parameters. I removed them previously and got a compilation error because of this.
I asked Matthieu about his preference and he said to just keep them.